agent/containers/images/visualizers/prom-graf-visualizer/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+/prometheus-${PROM_VERSION}.linux-amd64/prometheus --config.file=/prometheus.yml --storage.tsdb.path=/data </dev/null &>/dev/null &
```

Comment:
Perhaps have just one `run.sh` script which operates differently based on the environment variable settings?